### ISSUE TITLE
Can you do something for these old scripts ?

### ISSUE THREAD

**Issue by MaxDiesel7** [bombsquad-community/baport#11]
![image](https://github.com/user-attachments/assets/bf080063-a705-422e-8e09-92a26657d836)

can you port them ? i think they have api 4 ...
these are old games and custom character install scripts from the year 2017 or so ...
I can give it to you if you want

they came with a modded version of bombsquad called AGMP v3 by alligator, and bombdash [i dont remember the author's name] 

looking forward to your reply...

**Comment by brostosjoined** [comment]
Yeah am currently working on updating the script to support api below api 7. I contact you soon
